src/ui_fsmenu/options.cc

```diff
@@ -315,8 +316,15 @@ FullscreenMenuOptions::FullscreenMenuOptions(FullscreenMenuMain& fsmm,
 	language_dropdown_.selected.connect([this]() { update_language_stats(); });
 	training_wheels_reset_.sigclicked.connect([this]() {
 		training_wheels_reset_.set_enabled(false);
-		Profile training_wheels_profile;
-		training_wheels_profile.write(kTrainingWheelsFile);
+		cancel_.set_enabled(false);
+		apply_.set_enabled(false);
+		ok_.set_enabled(false);
```

Comment:
No need for that. While the TrainingWheelOptions is running, the buttons are inaccessible anyway.

Reply:
Yes they are, but this signals to the user that they are inaccessible. I find it more intuitive.